### PR TITLE
Checkstyle fixes, variable fixes, adjustments to model classes

### PR DESCRIPTION
**Pull request by TimYuenior**
Deleted: 
Shot.java in model package
With this, the whole model package is deleted.

Updates in PreviewController.java:
Checkstyle fixes
Made @FXML attributes private
Code Style changes (white spaces)
Made adjustments to work with the new Shot.java class in the script package.

### CODE REVIEW

File: src/main/java/nl/tudelft/contextproject/gui/PreviewController.java [hunk 1]
```diff
@@ -59,80 +61,112 @@ private void initialize() {
...
+        InstantPreset presetSix = new InstantPreset(dummySettings, 6);
+        
+        presetOne.setImageLocation("main/resources/placeholder_picture.jpg");
+        presetTwo.setImageLocation("main/resources/test2.jpg");
+        presetThree.setImageLocation("main/resources/test3.jpg");
+        presetFour.setImageLocation("main/resources/test4.jpg");
+        presetFive.setImageLocation("main/resources/test5.jpg");
+        presetSix.setImageLocation("main/resources/test6.jpg");
+
+
```
Comment (tkolenbrander) [inline src/main/java/nl/tudelft/contextproject/gui/PreviewController.java:92]: One vertical whitespace too many

File: src/main/java/nl/tudelft/contextproject/gui/PreviewController.java [hunk 2]
```diff
@@ -193,33 +231,37 @@ public int checkShots() {
         }
     }
     
+    /**
+     * Shows the shot that is selected.
+     * @param shot - The shot to be showed
+     */
     public void showShot(Shot shot) {
         if (checkShots() == 0) {
```
Comment (tkolenbrander) [inline src/main/java/nl/tudelft/contextproject/gui/PreviewController.java:263]: Maybe use a switch here for to keep it better structured.

**Comment by stevenmeijer** [comment]
Looks good after fixing Thomas' comments

**Comment by TimYuenior** [comment]
Made changes according to Thomas's feedback.

**Comment by tkolenbrander** [comment]
Looks good now! Let's merge.

**Pull request merged**